### ISSUE TITLE
GraphicalDashPluginCycleNextDash issue

### ISSUE THREAD

**Issue by NigeUK** [SHWotever/SimHub#1111]
**Describe the bug**
GraphicalDashPluginCycleNextDash (and equivalent previous Action)

**Simhub version**
Make sure to run the latest version
8.0.3

**To Reproduce**
1. Add GraphicalDashPluginCycleNextDash to a button on a graphical dash.
2. Add GraphicalDashPluginCyclePreviousDash to a button on a graphical dash.
3. Start Dash (I have tried web on host PC, and iPad
5. Buttons do not switch to next or previous Dash.
6. Repeat the above by clicking "test Dashboard" Dash and buttons work (i.e. advance to next or previous dashboard 

**Expected behavior**
Maybe I have misunderstood and these actions only relates to Dash testing.
I have a 'Main' dashboard which contains a sub-screen as a widget. The widget includes next\previous screen buttons, which work fine.
I tried to add multiple screens to the main dashboard and then add next/previous screen buttons. This results in the next/previous screen buttons on the widget changing the screen of the main dashboard, not the widget.
I assumed (mistakenly it seems) that a widget is treated as a completely separate entity, and therefore controls on the widget and the main screen would act independently.

Apologies if I've posted in the wrong area!

**Comment by SHWotever** [comment]
Hi ! You need to use next/previous screen options. GraphicalDashPluginCycleNextDash andits oposite ore only for templates in desktop mode.
![image](https://user-images.githubusercontent.com/2207331/177042509-fe32e037-c3ed-41f7-917a-3169e60137e4.png)


Those actions will only act based on the screen roles, if you are in game but all screens are only set to be "in game" simhub won't allow switching.

By default those actions only affects the main dash screens switching, to change a widget screen which indeed acts as a "sub screen" you need to assign first the actions which can do it in the designer (once again screens roles are important, and simhub won't show any screen not allowed in the current state (idle, or in game): 
![image](https://user-images.githubusercontent.com/2207331/177042691-91de6e4d-9f3d-469d-83c0-2d30b05184a3.png)

Overall there is a little additional subtility, if somehow you did create screens only for one mode, for instance only in game screens, simhub will take the first one even if it does not matches the current mode and prevent screen changing until you enter in race.

**Comment by NigeUK** [comment]
Thanks for your speedy reply!
That clarifies things somewhat.
I'm still not quite there though - I have set the NextScreenCommand and PreviousScreenCommand  options for the Widget to Action A and Action as per your screenshot.
I have set Next Page and previous page in Dash general settings to buttons on a Joystick (I also tried buttons on the dash).
I have set Action A and Action B to the buttons on the widget.
I  am in-game (playback) and all screens are set to idle and game.
I still get the same behaviour - the buttons on the widget change the main dashboard screen ( and the joystick buttons do the same).
Have I missed something?
![settings](https://user-images.githubusercontent.com/101148587/177044783-7f020239-1ce2-473d-b71b-2514d2ebe06c.png)
![controls](https://user-images.githubusercontent.com/101148587/177044784-c774304d-a6b4-4db2-917b-7a7278df365a.png)
